app/lib/controllers/session.js

```diff
@@ -112,6 +154,94 @@ Session.prototype.put = function (req, res, next) {
   return next()
 }
 
+/**
+ * Reset
```

Comment:
Why do we need method title? We haven't been using that and I can't find an example on [jsdoc.org](http://usejsdoc.org/index.html) with it.